src/test/components/App.test.js

```diff
@@ -0,0 +1,13 @@
+import '../localStorageMock';
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { BrowserRouter } from 'react-router-dom';
```

Comment:
for testing purposes you can also use [MemoryRouter](https://reacttraining.com/react-router/web/api/MemoryRouter)